test/file04.ts

```diff
@@ -0,0 +1,22 @@
+import styled from '@emotion/styled';
+import { Theme } from '@xsite-ui/common.style';
+
+export const Container = styled.div<{ theme?: Theme }>`
+  display: grid;
+  gap: ${({ theme }) => theme?.space?.[5] || 20}px;
```

Comment:
The use of optional chaining should be considered to handle the case where theme object is undefined or null.